xmodule/js/src/video/037_video_transcript_feedback.js

```diff
@@ -78,8 +78,43 @@
                         url: url,
                         type: 'GET',
                     })
-                    .success(function(data) {
-                        if (data.value === true) {
+                    .success(function(response) {
+                        console.log('Success: ', response.data);
+
+                        if (response.data.value === true) {
+                            this.markAsPositiveFeedback();
+                        } else {
+                            this.markAsNegativeFeedback();
+                        }
+                    });
+                },
+
+                setFeedbackForCurrentTranscript: function(feedbackValue) {
+                    var url = 'http://localhost:18760/api/v1/transcript-feedback';
+
+                    $.ajax({
+                        url: url,
+                        type: 'POST',
+                        dataType: 'json',
+                        // commenting this until rodri finishes the fetch for these 3 attributes
+                        // data: {
+                        //     transcript_language: this.currentTranscriptLanguage,
+                        //     video_uuid: this.videoUuid,
+                        //     user_uuid: this.userUuid,
+                        //     value: feedbackValue,
+                        // },
+
+                        data: {
+                            transcript_language: 'es',
+                            video_uuid: '6f437f8c-2287-49e0-8d5d-26fc87f038ac',
+                            user_uuid: '33d199ea-57f2-11ee-8c99-0242ac120002',
+                            value: feedbackValue,
+                        },
+                    })
+                    .success(function(response) {
+                        console.log('Success: ', response.data);
+
+                        if (feedbackValue === true) {
```

Comment:
Should we modify `this.markAsPositiveFeedback();` and `this.markAsNegativeFeedback();` so that it marks the correct button as selected but also unselect the one that was previously selected ? Also if we click on 👍  and 👍 was already selected, that will remove the feedback and we should update the UI accordingly (same with 👎 )